patch/kernel/archive/rockchip-rk3588-6.11/overlay/rockchip-rk3588-fanctrl.dtso

```diff
@@ -1,11 +0,0 @@
-/dts-v1/;
-/plugin/;
-/ {
-        fragment@0 {
-                target = <&pwm-fan>;
```

Comment:
Just use `&fan` instead of `&pwm-fan`.

Reply:
I successfully compiled 6.11 by making this change